scripts/example_download_dataset_split.py

```diff
@@ -0,0 +1,78 @@
+import argparse
+from concurrent.futures import ThreadPoolExecutor, as_completed
+from pathlib import Path
+
+import pandas as pd
+from tqdm import tqdm
+
+import paint.util.paint_mappings as mappings
+from paint import PAINT_ROOT
+from paint.data import StacClient
+from paint.util import set_logger_config
+
+set_logger_config()
+
+
+if __name__ == "__main__":
+    # Read in arguments.
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--input_dataset_splits",
+        type=str,
+        help="File containing the information on the dataset splits.",
+        default=f"{PAINT_ROOT}/benchmark_test/Benchmark_split-azimuth_train-10_validation-30.csv",
+    )
+    parser.add_argument(
+        "--output_dir",
+        type=str,
+        help="Path to save the extracted dataset splits.",
+        default=f"{PAINT_ROOT}/Benchmark_Datasets",
+    )
+    parser.add_argument(
+        "--filtered_calibration",
+        type=str,
+        help="List of calibration items to download.",
+        nargs="+",
+        default=["calibration_properties"],
+    )
+    args = parser.parse_args()
+
+    # Determine the name of the benchmark split being determined.
+    benchmark_name = Path(args.input_dataset_splits).name.split(".")[0]
+    base_output_dir = Path(args.output_dir)
```

Comment:
If we would set the default type to `pathlib.Path`, we could get rid of the conversions here, right?